cccs-build/superset/Dockerfile

```diff
@@ -1,7 +1,8 @@
 # Vault CA container import
 ARG VAULT_CA_CONTAINER=uchimera.azurecr.io/cccs/hogwarts/vault-ca:master_11376_a25c34e1
 FROM $VAULT_CA_CONTAINER AS vault_ca
-FROM uchimera.azurecr.io/cccs/superset-base:merge-cccs-2.0.1_20230215200331_b6275
+FROM uchimera.azurecr.io/cccs/superset-base:merge-cccs-2.0.1_20230223211519_b6312
+
```

Comment:
```suggestion
```
Not the end of the world, but we don't really need to add this extra whitespace line. Since it's not a blocker, I will approve anyway